api/users/models.py

```diff
@@ -243,6 +246,10 @@ def get_user_organisation(
                 % (self.id, getattr(organisation, "id", organisation))
             )
 
+    def get_user_roles(self):
+        user_roles = UserRole.objects.filter(user=self)
+        return user_roles
```

Comment:
If `settings.IS_RBAC_INSTALLED` is not present then this method will throw an error. Maybe catch the exception and fire a custom error to stop it from looking like a random 500 error?

Reply:
Done.